kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -0,0 +1,65 @@
+package com.kodilla.testing.collection;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class CollectionTestSuite {
+
+    @Before
+    public void before() {
+
+        System.out.println("Start testu Collection ");
+    }
+
+    @After
+    public void after() {
+        System.out.println("Koniec testu Collection ");
+
+    }
+
+    @Test
+    public void testOddNumbersExterminatorEmptyList() {
+//given
+        ArrayList<Integer> numbers = new ArrayList<Integer>();
+
+        OddNumbersExterminator packNumbers = new OddNumbersExterminator();
+        packNumbers.exterminate(numbers);
+
+//when
+        ArrayList<Integer> listEven = packNumbers.getListEven();
+//then
+        Assert.assertTrue(numbers.isEmpty() && listEven.isEmpty());
```

Comment:
You should NOT combine 2 conditions in one assertion - if test fail, you won't know which condition caused that.